subsys/bluetooth/host/settings.c

```diff
@@ -137,7 +137,7 @@ static int set(const char *name, size_t len_rd, settings_read_cb read_cb,
 		if (len < sizeof(bt_dev.id_addr[0])) {
```

Comment:
@jhedberg While going over ssize_t and zd change I noticed this comparison. This is ssize_t vs size_t comparison.
If I enable `-Wsign-compare` I get a worning on this line (and similar for IRK).
Should we fix this one as well?

Reply:
I would definitely fix it

Reply:
Noted. Will not fix as part of this PR though.